src/pages/organizers/[organizerId]/preview/RequestOwnershipModal.tsx

```diff
@@ -31,7 +32,9 @@ const RequestOwnershipModal = ({
   const { t, i18n } = useTranslation();
   const queryClient = useQueryClient();
   const router = useRouter();
-  const organizerId = router.query.organizerId as string;
+  const organizerIdFromQuery = router.query.organizerId as string;
+  const organizerIdFromOffer = parseOfferId(organizer?.['@id'] ?? '');
+  const organizerId = organizerIdFromQuery ?? organizerIdFromOffer;
```

Comment:
I would inline organizerIdFromQuery and organizerIdFromOffer directly into the organizerId assignment. Since they’re only used once and not reused later, creating separate variables just adds extra lines without improving clarity, because other developers don't know if this variable will be re-used later ot not.

Reply:
```suggestion
const organizerId = (router.query.organizerId as string) ?? parseOfferId(organizer?.['@id'] ?? '');
```